src/globals/js/misc/on-focus-by-keyboard.js

```diff
@@ -0,0 +1,52 @@
+const hasFocusout = 'onfocusout' in window;
+const focusinEventName = hasFocusout ? 'focusin' : 'focus';
+const focusoutEventName = hasFocusout ? 'focusout' : 'blur';
+
+/**
+ * Differentiate between keyboard and mouse-triggered focusout/blur events
+ * @param {Element} node  The element to attach event listeners to
+ * @param {string} name The event name to listen to
+ * @param {Function} callback The callback function to invoke
+ * @returns {Handle} The handle to release the attached event handler
+ */
+export default function onFocusByKeyboard(node, name, callback) {
+  /**
+   *
```

Comment:
Nitpick - Do we want to update the comment here...?

Reply:
updated!